nuxeo-full-ps-template.yaml

```diff
@@ -0,0 +1,1989 @@
+          - name: nuxeolog
+            mountPath: /var/log/nuxeo
+          - name: nuxeotmp
+            mountPath: /opt/nuxeo/server/tmp
+        volumes:
+        - name: ${APPLICATION_NAME}-binaries
+          persistentVolumeClaim:
+            claimName: ${APPLICATION_NAME}-dev-binaries
+        - name: nuxeoconf
+          configMap:
+            name: ${APPLICATION_NAME}-dev-conf
+        - name: connect-secret
+          secret:
+            secretName: ${APPLICATION_NAME}-connect
+        - name: nuxeodata
+          emptyDir: {}
+        - name: nuxeolog
+          emptyDir: {}
+        - name: nuxeotmp
+          emptyDir: {}
+        dnsPolicy: ClusterFirst
+        restartPolicy: Always
+        terminationGracePeriodSeconds: 30
+    test: false
+    triggers:
+    - type: ConfigChange
+    - type: "ImageChange"
+      imageChangeParams:
+        automatic: false
+        containerNames:
+        - ${APPLICATION_NAME}-appserver
+        from:
+          kind: ImageStreamTag
+          name: ${APPLICATION_NAME}-nuxeo:latest
+
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: ${APPLICATION_NAME}-dev-nuxeo
+    labels:
+      app: ${APPLICATION_NAME}-dev
+      component: nuxeo
+      role: lb
+  spec:
+    clusterIP: None
+    selector:
+      app: ${APPLICATION_NAME}-dev
+      component: nuxeo
+      stage: dev
+    ports:
+    - name: 8080-tcp
+      port: 8080
+      protocol: TCP
+
+- apiVersion: v1
+  kind: Route
+  metadata:
+    labels:
+      app: ${APPLICATION_NAME}-dev
+    name: ${APPLICATION_NAME}-dev-nuxeo-route
+  spec:
+    host: ${APPLICATION_NAME}-dev.${DOMAIN_SUFFIX}
+    tls:
+      termination: edge
+      insecureEdgeTerminationPolicy: Redirect
+    to:
+      kind: Service
+      name: ${APPLICATION_NAME}-dev-nuxeo
+    port:
+      targetPort: 8080-tcp
+    wildcardPolicy: None
+
+############################################################################################################
+# UAT Deployment
+############################################################################################################
+
+
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: ${APPLICATION_NAME}-uat-conf
+    labels:
+      app: ${APPLICATION_NAME}-uat
+      role: config
+      stage: dev
+  data:
+    nuxeo.conf: |
+      # Additional nuxeo.conf parameters
+    init.sh: |
+      #!/bin/sh
+
+      if [ ! -f $NUXEO_DATA/instance.clid -a -f /opt/nuxeo/connect/connect.properties ]; then
+        . /opt/nuxeo/connect/connect.properties
+        if [ -n "$NUXEO_CONNECT_USERNAME" -a -n "$NUXEO_CONNECT_PASSWORD" -a -n "$NUXEO_STUDIO_PROJECT" ]; then
+          echo "---> Registering instance on connect"
+          nuxeoctl register $NUXEO_CONNECT_USERNAME $NUXEO_STUDIO_PROJECT dev openshift $NUXEO_CONNECT_PASSWORD
+        fi
+      fi
+
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    annotations:
+      volume.beta.kubernetes.io/storage-class: shared
```

Comment:
Now it's possible to replace the annotation with the `storageClass` property of a PVC (see https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistent-volumes)

Reply:
done